portfolio/src/main/webapp/script.js

```diff
@@ -43,11 +43,48 @@ function displayAbout() {
     });
 }
 
+function displayGame() {
+    const title = document.getElementById('content-title');
+    title.innerHTML = "Let's play a fun game (well, at least I find it fun <i class=\"em em-smile\" aria-role=\"funny\" aria-label=\"Smile\"></i>)";
+    $(function(){
+        $("#content-panel").load("main_page_clickables/game.html"); 
+    });   
+}
+
+function displayCountry() {
+    console.log("Getting the country name country");
+
+    const responsePromise = fetch('/data');
+
+    responsePromise.then(handleResponse);
```

Comment:
btw, promises have not just a `.then(..)` method but also error callback (`.catch( ...)`). We can skip it for the portfolio, but just be aware that HTTP requests can (And often do!) fail :)